build/msw/makefile.vc

```diff
@@ -5742,6 +5754,9 @@ __webview_additional_libdirs_arch_FILENAMES = x86
 !if "$(TARGET_CPU)" == "AMD64"
 __webview_additional_libdirs_arch_FILENAMES = x64
 !endif
+!if "$(TARGET_CPU)" == "ARM"
+__webview_additional_libdirs_arch_FILENAMES = arm
+!endif
```

Comment:
Rebaking removes this part (and the same addition below) and I don't immediately see how to fix this. AFAICS it doesn't really matter because webview is not supported on ARM32 (at least I know that Chromium doesn't support it).

Reply:
Probably it doesn't matter, no. Now the build should at least behave sensibly when compiling without extra components enabled.

I have now squashed my fixes and adjusted the commit message a little. Should be ready to merge after rebaking.